tailscale/resource_logstream_configuration.go

```diff
@@ -70,6 +70,26 @@ func resourceLogstreamConfiguration() *schema.Resource {
 				Optional:    true,
 				Sensitive:   true,
 			},
+			"upload_period_minutes": {
+				Type:        schema.TypeInt,
+				Description: "The number of minutes to wait in between uploading new logs. If the quantity of logs does not fit within a single upload, multiple uploads will be made. Defaults to 1.",
+				Default:     1,
```

Comment:
@mpminardi Will specifying defaults here (or in the API implementation) cause spurious plan updates for existing configurations that don't specify these parameters?